docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -352,17 +353,17 @@ kubectl logs ${CONTROLLER_POD} -n ${ACK_K8S_NAMESPACE}
 CONTROLLER_DEPLOYMENT=$(kubectl get deploy -n ${ACK_K8S_NAMESPACE} -o jsonpath='{.items..metadata.name}')
 kubectl edit deploy/${CONTROLLER_DEPLOYMENT} -n ${ACK_K8S_NAMESPACE}
 ```
-This is how your values should look after change.
+This is how your values should look after changes are applied.
 ```
         - --aws-region
         - $(AWS_REGION)
         - --aws-endpoint-url
         - $(AWS_ENDPOINT_URL)
-        **- --enable-development-logging
+        - --enable-development-logging
         - "true"
         - --log-level
-        - debug**
+        - debug
         - --resource-tags
         - $(ACK_RESOURCE_TAGS)
 ```
```

Comment:
Hmm I guess we didn't make the changes in the original PR, but really we shouldn't be touching deployments directly. Only by updating the Helm chart values and redeploying.

Reply:
This is mainly for troubleshooting purpose. I see end users using `kubectl` and not using `helm` for troubleshooting. Had a discussion with Amine on this [here](https://github.com/aws-controllers-k8s/community/pull/1453#pullrequestreview-1098158023)